OxfordBodleian/Juel-Jensen/BDLaethd20.xml

```diff
@@ -159,29 +3591,82 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    elements of punctuation signs, numerals and text dividers,
+                                    holy names (<persName ref="PRS6819Mary"/>, <persName ref="PRS4377Gabriel"/>, 
+                                    rarely <persName ref="PRS5684JesusCh"/>), the names of <persName ref="PRS12911KidanaK"/>,
+                                    <persName ref="PRS13031WaldaA"/>, <persName ref="PRS12964Gorgoryos"/>. 
+                                    <persName ref="PRS13026KenfaM"/>, captions of miniatures.
+                                </seg>
                             </handNote>
                         </handDesc>
                         
                        <!-- <decoDesc>
                          
                         </decoDesc>-->
                         
-                     <!--   <bindingDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="#5r"/>
+                                    <desc type="Inventory">List of liturgical vessels and books belonging to a church of  Madḫāne ʿĀlam.</desc>
+                                    <q xml:lang="am">ኍልቈ፡ ንዋዮ፡ ቅድሳት፡ ዘመድኃኔ፡ ዓለም፡
+                                        <persName ref="PRS12911KidanaK"><roleName type="title">አባ፡</roleName> ኪዳነ፡ ክርስቶስ፡</persName> የሰጡ፡
+                                    ፫መስቀል፡ ፫ጽና፡ ፩ናስ፡ የደም፡ ጽዋ፡
+                                    ፩ጠርሙስ፡ ፩ቃጭል፡ ፩የናስ፡ ማብርያ፡
+                                    ፩የፍም፡ መጫርያ፡ መረዋት፡
+                                    ፩ጁህ፡ በርኖስ፡ ፩ማኅፈድ፡ ፩<ref type="work" corresp="LIT1560Gospel">ወንጌል፡</ref>
+                                        ፩<ref type="work" corresp="LIT1960Mashaf">ቅዳሴ፡</ref> ፩<ref type="work" corresp="LIT2384Taamme">ተአምረ፡ እግዝእትነ፡ ማርያም</ref>፡ ፩መዝገብ፡ ፩የብራና፡ ሥዕል፡
+                                    ፪ቤት፡ ያሹሩት፡ በድሙሩ፡ ፳፩ነው፡
+                                    ይህነን፡ የተጻፈውን፡ የሸጠና፡ የሰረቀ፡ የፋቀ፡ በሥልጣነ፡ ጴጥሮ<add place="above">ስ</add>፡ ወጳውሎስ፡
+                                    ውጉዘ፡ ይኩንx x x
+                                    </q>
+                                </item>
+                                
+                                <item xml:id="a2">
+                                    <locus target="#5r"/>
+                                    <desc type="Inventory">Further list of objects, separated from
+                                    <ref target="#a1"/> by a fine line.</desc>
+                                    <q xml:lang="am">በ፯ወቄት፡ ከ፫ድሪም፡ የጕልላት፡ መስቀል፡ የተሰራ፡ ቍጽሩ፡
+                                    ፲፱ነው፡ ከቃቤት፡ አለ፡</q>
+                                </item>
+                                
+                                <item xml:id="a3">
+                                    <locus target="#7r"/>
+                                    <desc type="OwnershipNote">Note stating that this manuscript belonged to 
```

Comment:
I would add that the note is written in pale black ink in the upper corner of the folio.